<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plooter</class>
 <widget class="QMainWindow" name="Plooter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Plooter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="spacer9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="spacer3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spacer1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gradePush">
        <property name="text">
         <string>Grade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gradeContent"/>
      </item>
      <item>
       <widget class="QPushButton" name="insertPush">
        <property name="text">
         <string>Insert:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="insertContent"/>
      </item>
      <item>
       <widget class="QPushButton" name="deletePush">
        <property name="text">
         <string>Delete:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deleteContent"/>
      </item>
      <item>
       <widget class="QPushButton" name="clearPush">
        <property name="text">
         <string>Clear!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dataType">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>int</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>char</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spacer4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spacer2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="spacer10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gradeLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spacer12">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="spacer8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spacer6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="imageTree"/>
      </item>
      <item>
       <widget class="QLabel" name="spacer5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spacer7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="spacer11">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
